service/src/test/java/bio/terra/lz/futureservice/pact/consumer/SamServiceConsumerPactTest.java

```diff
@@ -0,0 +1,187 @@
+package bio.terra.lz.futureservice.pact.consumer;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactConsumerTestExt;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.PactSpecVersion;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import bio.terra.common.iam.BearerToken;
+import bio.terra.landingzone.library.configuration.LandingZoneSamConfiguration;
+import bio.terra.landingzone.service.iam.LandingZoneSamClient;
+import bio.terra.landingzone.service.iam.LandingZoneSamService;
+import bio.terra.landingzone.service.iam.SamConstants;
+import bio.terra.lz.futureservice.app.service.status.SamStatusService;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@Tag("pact-test")
+@ExtendWith(PactConsumerTestExt.class)
+public class SamServiceConsumerPactTest {
+  private static final String RESOURCE_TYPE = "resourceType";
+  private static final String ACTION = "action";
+  private static final String RESOURCE_ID = "5739d704-b0f7-11ee-9f52-00155df9466d";
+  private static final String SAM_LISTRESOURCESANDPOLICIESV2_RESPONSE_BODY =
```

Comment:
Just a note, that whatever we put in the pact should be the _minimal_ response the service needs from Sam (I don't know if this is or not).

Reply:
Yep. We need only resourceId. The rest is not used. Will clean it up.